CHANGELOG.adoc

```diff
@@ -17,6 +17,7 @@
 === New Features
 
 - https://github.com/eclipse-sirius/sirius-components/issues/1567[#1567] [project] Add support for project templates
+- https://github.com/eclipse-sirius/sirius-components/issues/1463[#1463] [diagram] Add support for the direct edit of the start and end labels of an edge
```

Comment:
Maybe mention than for these label the direct-edit can only be triggered by a double-click? I was confused at first on how to trigger their edition and it seems to be the only way.